src/qz/utils/ConnectionUtilities.java

```diff
@@ -0,0 +1,29 @@
+package qz.utils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.net.URLConnection;
+
+import qz.common.Constants;
+
+public final class ConnectionUtilities {
+    private ConnectionUtilities() {
+    }
+
```

Comment:
Since the function name is identical to that of `URLConnection` please add a comment explaining the purpose.